.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Should we do this less frequent, say weekly? Or monthly? I am assuming this will constantly cut new PRs once a new GH action version has been released?

Reply:
I changed it to weekly. 
Currently the dependencies are set to major versions, so after the first bunch of updates there will only be a PR like once a year or so.